### PR TITLE
(maint) Gemfile updated

### PR DESCRIPTION
**Pull request by petems**

### CODE REVIEW

**Comment by igalic** [comment]
@petems shouldn't these changes go into .sync.yml?

**Comment by bmjen** [comment]
Travis-CI is failing due to missing rspec-system in Gemfile.

~~~
[~/proj/puppetlabs-ruby]$ bundle exec rake validate_puppet_syntax
rake aborted!
LoadError: cannot load such file -- rspec-system/rake_task
/Users/bmjen/proj/puppetlabs-ruby/Rakefile:3:in `require'
/Users/bmjen/proj/puppetlabs-ruby/Rakefile:3:in `<top (required)>'
(See full trace by running task with --trace)
~~~

**Comment by petems** [comment]
@igalic Probably, but I don't think this repo is under modulesync control yet

**Comment by petems** [comment]
@bmjen Fixed! :+1:

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: ruby
-bundler_args: --without development
-before_script: 
+bundler_args: --without development system_tests
+before_script:
   - bundle exec rake validate_puppet_syntax
```
Comment (bmjen) [inline .travis.yml]: @petems since you removed these tasks from the Rakefile, you'll need to remove them from the travis.yml.

**Pull request merged**